src/gs-page.c

```diff
@@ -246,6 +250,36 @@ gs_page_app_removed_cb (GObject *source,
 	}
 }
 
+static void
+gs_page_app_copied_cb (GObject *source,
+		       GAsyncResult *res,
+		       gpointer user_data)
+{
+	g_autoptr(GsPageHelper) helper = (GsPageHelper *) user_data;
+	GsPluginLoader *plugin_loader = GS_PLUGIN_LOADER (source);
+	GsPage *page = helper->page;
+	gboolean ret;
+	g_autoptr(GError) error = NULL;
+
+	ret = gs_plugin_loader_job_action_finish (plugin_loader, res, &error);
+	if (g_error_matches (error,
+			     GS_PLUGIN_ERROR,
+			     GS_PLUGIN_ERROR_CANCELLED)) {
+		g_debug ("%s", error->message);
```

Comment:
Maybe this message needs a bit more context to be useful.

Reply:
I prefixed with "app copy cancelled: "